test/src/module/command/archivePushTest.c

```diff
@@ -292,12 +291,17 @@ testRun(void)
 
         TEST_ERROR(cmdArchivePush(), ArchiveDuplicateError, "WAL file '000000010000000100000001' already exists in the archive");
 
-        // Save it to a new file instead
-        argListTemp = strLstDup(argList);
-        strLstAddZ(argListTemp, "pg_wal/000000010000000100000002");
+        // -------------------------------------------------------------------------------------------------------------------------
+        TEST_TITLE("WAL with absolute path and no pg1-path");
+
+        argListTemp = strLstNew();
+        strLstAddZ(argListTemp, "--" CFGOPT_STANZA "=test");
+        // strLstAdd(argListTemp, strNewFmt("--" CFGOPT_PG1_PATH "=%s/pg", testPath()));
```

Comment:
I don't think you need to leave this commented out code here - the title of the test explains what is happening.

Reply:
That's just there for my testing. This isn't quite a finished commit, just wanted to float the general idea.